typescript-deno/src/mod.ts

```diff
@@ -0,0 +1,3 @@
+const result = false;
+
```

Comment:
It would be nice to have a console.log here just so when you run `deno run src/mod.ts` it's clearer that you've successfully run the code.